recipes/miniaudio/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import ConanFile, tools
+import os
+import glob
+
+
+class MiniaudioConan(ConanFile):
+    name = "miniaudio"
+    description = "A single file audio playback and capture library."
+    topics = ("conan", "miniaudio", "header-only", "sound")
+    homepage = "https://github.com/mackron/miniaudio"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = ["Unlicense", "MIT-0"]
+    settings = "os", "compiler", "build_type", "arch"
+
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob(self.name + "-*/")[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="miniaudio.h", dst="include", src=self._source_subfolder)
+        self.copy(
+            pattern="stb_vorbis.c",
+            dst="include/extras",
+            src=os.path.join(self._source_subfolder, "extras"),
+        )
+        self.copy(
+            pattern="README.md",
+            dst="licenses",
+            src=os.path.join(self._source_subfolder, "extras/speex_resampler/"),
+        )
```

Comment:
As an enterprise consumer, I would have a bad day if I found this checked into a production build...
➡️ Please add all the license to the recipe attribute! 
- BSD-3-Clause https://github.com/mackron/miniaudio/blob/d06d4983d3ff512690eb37f5edd25bf96f8f3764/extras/speex_resampler/thirdparty/speex_resampler.h#L1
- BSD-2-Clause you have from the README

I saw a comment about it https://github.com/conan-io/conan-center-index/pull/4910#issuecomment-800195702
But it's unclear why you are including this... it looks like a sample.

If you do not need could it be excluded from the package?

Reply:
these are 3rd-parties, not the `miniaudio` itself. so they are dependencies, and BSD should be specified in the dependency conanfile, not in the miniaudio (which is Unilicense)

Reply:
This is an opt-in "addon" to `miniaudio`. It's inclusion forces the license from Unlicense to MIT-0. The `readme` file @prince-chrismc highlighted is pretty much the opus license: 
https://github.com/mackron/miniaudio/tree/master/extras/speex_resampler 
And it's not a sample, it's a resampler, it's an essential functionality of the library (which already includes one, but with a poorer algorithm).

You're the experts, but I would assume we only have to ship the licenses from `miniaudio` itself, aka Unlicense and MIT-0. In that regard, that file shouldn't be needed, but an MIT-0 file is. 

I'll hold for your suggestions.

Reply:
Okay, the `LICENSE` file in the root of the repo includes both licenses. That makes the file that worried @prince-chrismc useless, and I removed it. And as @SSE4 said, what is in the headers of those "dependencies" of this library shouldn't be of concern, since we're consuming the library as a whole, with a compatible license.